sns/cm/repair/ut/cm.c

```diff
@@ -68,6 +68,7 @@ static struct m0_fom_simple     iter_fom;
 static struct m0_fom_timeout    iter_fom_timeout;
 static struct m0_semaphore      iter_sem;
 static const struct m0_fid      M0_SNS_CM_REPAIR_UT_PVER = M0_FID_TINIT('v', 1, 8);
+static enum m0_cm_op 			op;
```

Comment:
Spacing. op should be aligned with M0_SNS_CM_REPAIR_UT_PVER.

Reply:
Done in latest commit